config.yml

```diff
@@ -1,40 +1,67 @@
 free_world: world
+# world imprisoned players respawn to once freed
 free_respawn: true
+# if set to false: sends pearled players to free world if freed when offline. if set to true: keeps players in end if freed when offline.
 free_tppearl: true
+# if freed while online, will teleport to the player holding the pearl.
 damagelog_min: 3
+# unsure
 damagelog_ticks: 600
+# unsure
 ppkill_enabled: true
+# enable pearl holders to use /ppkill while a player is summoned to kill them if true.
 prison_stealing: true
+# enable another player to repearl a summoned player if true
 prison_musthotbar: true
+# requires players to have enderpearl in hotbar to pearl another killed player
 summon_damage_radius: 20
+# the distance a summond player can travel by default from the pearlholder
 summon_damage_ticks: 20
+# how many ticks until damage is dealt to pearled player outside summon_damage_radius
 summon_damage_amt: 2
+# amount of damage dealt to prisoned player
 prison_world: world_the_end
+# which world players are sent to when killed by another player with an enderpearl in their hotbar (or inventory depending on prison_hotbar setting)
 prison_resetbed: false
+# unsure
 chunk_delay: 1200
```

Comment:
The amount of time to wait if a pearl is dropped due to logging out and the chunk is trying to unload to free a pearl.